lizard_auth_server/models.py

```diff
@@ -60,14 +100,14 @@ class Portal(models.Model):
         verbose_name=_('shared secret'),
         max_length=64,
         unique=True,
-        default=gen_key('Portal', 'sso_secret'),
+        default=GenKey('Portal', 'sso_secret'),
```

Comment:
Ik zie geen custom `save` methode voor `Portal`. Hij heeft wel een `rotate_keys` methode, maar die wordt nergens in lizard-auth-server aangeroepen.